.ci/azure-pipelines.yml

```diff
@@ -100,48 +92,55 @@ jobs:
      
 
     - script: |
-        type $(System.DefaultWorkingDirectory)\server\v212\aisol\bin\winx64\log.txt
+        type $(System.DefaultWorkingDirectory)\server\v$(ANSYS_VERSION)\aisol\bin\winx64\log.txt
       displayName:  'Show DPF Server Logs'
-      condition: always() 
+      condition: always()
+      continueOnError: true
     
     - template: templates\kill-servers-windows.yml   
-      
+
+
+- job: WindowsAnsys2021R2
+  variables:
+    python.version: '3.8'
+    ANSYS_VERSION: 212
+    DISPLAY: ':99.0'
+    PYANSYS_OFF_SCREEN: True
+    DPF_PORT: 32772
+    vstsPackageVersion: '21.2.3'
+  pool:
+    vmImage: 'windows-2019'
+
+  steps:
+    - template: templates\prepare-environment-windows.yml
+
+    - script: |
+            pip install ansys-grpc-dpf==0.3.0
+      displayName: Install proper version of ansys-grpc-dpf
```

Comment:
This shouldn't be necessary and should be dictated in `install_requires` in `setup.py`.